src/requester.ts

```diff
@@ -0,0 +1,89 @@
+import bsv from 'bsv';
+import { AuthHeader, setSignature } from './authentication';
+import { Logger } from './logger';
+import { ErrorNoAdminKey, ErrorResponse } from './errors';
+
+export class Requester {
+  private logger: Logger;
+
+  private adminKey?: bsv.HDPrivateKey;
+  private signingKey?: bsv.HDPrivateKey | bsv.PrivateKey;
+  private xPubString?: string;
+  private url: string;
+
+  static CreateSigningRequester(
```

Comment:
Can it be lowercase?

Reply:
I've completely removed the static methods.